atomic_reactor/inner.py

```diff
@@ -174,6 +174,7 @@ def __init__(self, uri, insecure=False):
         """
         super(DockerRegistry, self).__init__(uri, insecure=insecure)
         self.digests = {}  # maps tags (str) to their digest, if available
+        self.config = None  # stores image config from the registry
```

Comment:
Media type for this is `application/vnd.docker.container.image.v1+json` -- can we include that in the comment so we know exactly what this is intended to be?